lightninglabs/lightning-terminal
# chore: remove JS code generated from protos

The JS/TS code generated by `protoc` from the `*.proto` files do not need to be stored in the repo. They can be generated after cloning locally. This PR removes the generated files and updates the `build-protos.sh` script to fix a small typing error.

To test:
 - delete the `/app/src/types/generated/` dir
 - run `cd app; yarn protos` to generate the JS/TS files
 - confirm no errors are displayed and new files are in `/app/src/types/generated/`
 - run `yarn lint; yarn tsc; yarn test:ci`
 - confirm no errors
